### ISSUE TITLE
Issuing codes might report the wrong date in `user_stats`.

### ISSUE THREAD

**Issue by jeremyfaller** [google/exposure-notifications-verification-server#819]
### TL;DR

If you issue a code, the `realm_stats` table in the database might record an incorrect date. See attached screen shot:

![image](https://user-images.githubusercontent.com/3063869/96022843-800c5900-0e1f-11eb-8d74-57894749f173.png)

**Expected behavior**

I expect all the dates to line up

**Observed behavior**

See screenshot

### Reproduction

1.  create new db, start server
2.  issue a few codes
3.  notice the dates are wrong???  (This step MIGHT not work, but it's the best I got.)

**Screenshots**

![image](https://user-images.githubusercontent.com/3063869/96022918-9dd9be00-0e1f-11eb-9f93-22b898fb0e4f.png)

**Environment**

- OS:
- Browser:
- Version:

**Additional information**

**Comment by jeremyfaller** [comment]
/assign

Gonna take a look at this, I think I have an idea what it is.
